db/migrate/20250723095917_add_session_token_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddSessionTokenToUsers < ActiveRecord::Migration[7.0]
+  def change
+    add_column :users, :session_token, :string
```

Comment:
```suggestion
    add_column :users, :remember_digest, :string
```
Đúng thì mình sẽ tạo column remember_digest này để dùng được trong chapter 9 luôn e nhé

Reply:
Fix tương tự cho file migrate còn lại